src/pricing/calculations/extPaint.js

```diff
@@ -0,0 +1,35 @@
+import { floorsDiscount } from '../content/extPaint/categories'
+import { overHangRatio } from '../content/extPaint/categories'
+import { overHang } from '../content/extPaint/categories'
+import { windowPrice } from '../content/extPaint/categories'
```

Comment:
Maybe you haven't seen my last comment (on the `calculations/roof.js` also), you can write:
`import {floorsDiscount, overHangRatio, overHang, windowPrice} from '../content/extPaint/categories'`